app/form_models/jobseekers/subscription_form.rb

```diff
@@ -20,7 +35,10 @@ def initialize(params = {}) # rubocop:disable Metrics/AbcSize,Metrics/Cyclomatic
     @frequency = params[:frequency]
     @keyword = params[:keyword] || search_criteria[:keyword]
     @location = params[:location] || search_criteria[:location]
-    @job_roles = params[:job_roles]&.reject(&:blank?) || search_criteria[:job_roles] || []
+    @teaching_job_roles = params[:teaching_job_roles]&.reject(&:blank?) || search_criteria[:teaching_job_roles] || []
+    @teaching_support_job_roles = params[:teaching_support_job_roles]&.reject(&:blank?) || search_criteria[:teaching_support_job_roles] || []
+    @non_teaching_support_job_roles = params[:non_teaching_support_job_roles]&.reject(&:blank?) || search_criteria[:non_teaching_support_job_roles] || []
+    @visa_sponsorship_availability = params[:visa_sponsorship_availability_options]&.reject(&:blank?) || search_criteria[:visa_sponsorship_availability]
```

Comment:
Are we also adding the visa sponsorship availability to the subscriptions within this PR? Was it missing or something?

Reply:
Oh I see this was added in a specific commit. So defo not by mistake. Question resolved!